### PR TITLE
Refactor color and pos in menu_item

### PR DESCRIPTION
**Pull request by dmentipl**
We define:

- `RED`, `GREEN`, `BLACK` as tuples specifying a color, e.g. `BLACK = (0, 0, 0)`, which can be passed into `color565`
- `POS` is a list of tuples specifying positions for the menu items, e.g. `POS[0] = (45, 20)`

### CODE REVIEW

File: pico_menu_rotary_test.py [hunk 1]
```diff
@@ -50,25 +58,33 @@
...
+display.fill_rectangle(0, 0, 240, 320, color=color565(0,0,0))
+
+def menu_item(color, pos, font, Text):
+    """Menu item.
+
+    Args:
+        color: Color as a tuple (c1, c2, c3).
+        pos: Position as a tuple (x, y).
+        font: The font.
+        Text: The text to display.
```
Comment (dmentipl) [inline pico_menu_rotary_test.py:70]: This docstring uses Google-style. See [here](https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html) for more.

File: pico_menu_rotary_test.py [hunk 2]
```diff
@@ -33,6 +33,14 @@
 
 # fonts = [glcdfont,tt14,tt24,tt32]
 
+# Colors
+RED = (255, 0, 0)
+GREEN = (0, 255, 0)
+BLACK = (0, 0, 0)
```
Comment (dmentipl) [inline pico_menu_rotary_test.py:39]: Each color is a tuple of integers between 0 and 255.

File: pico_menu_rotary_test.py [hunk 3]
```diff
@@ -33,6 +33,14 @@
...
+
+# Pos
+POS = [(45, 20), (45, 60), (45, 100), (45, 140)]
```
Comment (dmentipl) [inline pico_menu_rotary_test.py:42]: There are four positions each with the same x value (45) but different y values.

**Comment by dmentipl** [comment]
@fmentiplay, just an update: I added your changes, i.e. adding the library files and updating the state diagram image, to this PR.

**Comment by dmentipl** [comment]
@fmentiplay: would you like to review this PR?

**Comment by fmentiplay** [comment]
Thanks for the suggestions.

**Pull request merged**